sdk/cosmos/azure-cosmos-kafka-connect/doc/configuration-reference.md

```diff
@@ -2,64 +2,64 @@
+| `azure.cosmos.account.key`                                        | `""`         | Cosmos DB Account Key (only required in case of `auth.type` as `MasterKey`)                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| `azure.cosmos.auth.aad.clientId`                                  | `""`         | The clientId/ApplicationId of the service principal. Required for `ServicePrincipal` authentication.                                                                                                                                                                                                                                                                                                                                                                                           |
+| `azure.cosmos.auth.aad.clientSecret`                              | `""`         | The client secret/password of the service principal.                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| `azure.cosmos.mode.gateway`                                       | `false`      | Flag to indicate whether to use gateway mode. By default it is false.                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| `azure.cosmos.region.preferredList`                               | `[]`         | Preferred regions list to be used for a multi region Cosmos DB account. This is a comma separated value (e.g., `[East US, West US]` or `East US, West US`) provided preferred regions will be used as hint. You should use a collocated kafka cluster with your Cosmos DB account and pass the kafka cluster region as preferred region. See list of azure regions [here](https://docs.microsoft.com/dotnet/api/microsoft.azure.documents.locationnames?view=azure-dotnet&preserve-view=true). |
```

Comment:
we should revert this back to `azure.cosmos.peferredRegionsList`

Reply:
reverted